commons/src/main/java/gov/cms/qpp/conversion/model/error/ErrorCode.java

```diff
@@ -130,7 +130,8 @@
 			"on Sub-population `(Sub Population)` for eCQM measure `(Measure Id)`", true),
 	CPC_PLUS_SUBMISSION_ENDED(68, "CPC+ Submission is after the end date `(Submission end date)`", true),
 	INVALID_PERFORMANCE_PERIOD_FORMAT(69, "`(Performance period start or end date)` is an invalid date format. "
-		+ "Please use the format 'yyyyMMdd'. Ex.: '20170101'", true);
+		+ "Please use a standard ISO date format. "
+		+ "Possible valid values are 2017-02-26, 2017/02/26T01:45:23, or 2017-02-26.", true);
```

Comment:
You have two of the same example here.  Also, I would call out these as examples.  I am afraid that users may mistake that these are the only valid values.